xmpp.go

```diff
@@ -1,4 +1,7 @@
-// Copyright 2011 The Go Authors.  All rights reserved.
+// Copyright 2011 The Go Authors.
+// Copyright 2018 Siemens AG
```

Comment:
if you want to reserve this patch, I can't merge this.

Reply:
i have reverted that, was a try to match the company rules. Hope that this is now ok.